src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -57,6 +57,8 @@ public function load(array $configs, ContainerBuilder $container)
             }
         }
 
+        $container->setAlias('api_platform.naming.path.default', $config['naming']['path']);
```

Comment:
It's used for injecting the service into **api_platform.route_loader**, but I misnamed it, it should be just **api_platform.naming.path**